app/actions/CommonActions.scala

```diff
@@ -62,6 +62,29 @@ object CommonActions {
       case _ => _platform
     }
 
+    def playId: Option[String] = {
+      for {
+        playSessionCookie <- request.cookies.get("PLAY_SESSION")
+        value <- Option(playSessionCookie.value)
+      } yield {
+        value.take(value.indexOf("-"))
+      }
+    }
+
+    def initialSessionId: String = {playId.getOrElse("")}
+
+    def sessionId: String = {
```

Comment:
If we don't need to set the `payment_session` value (see comment below), then this method can be removed completely, and `initialSessionId` renamed to `sessionId`.